src/client/js/CoreEditor.js

```diff
@@ -261,16 +297,18 @@ class CoreEditor extends React.Component {
                     editorState={editorState}
                     onToggle={this.toggleInlineStyle}
                   />
-                  {!this.state.showURLInput ? null :
+                  <Modal shouldCloseOnOverlayClick={true} style={customStyles} isOpen={this.state.showURLInput == 1}>
+                  
                     <LinkUrlControls
```

Comment:
@muddaserahmed I believe that best is move modal inside LinkUrlControls class, and rename LinkUrlConrols to LinkUrlDialog or something like that